src/Raven.Studio/typescript/models/database/tasks/periodicBackup/serverWideBackupConfiguration.ts

```diff
@@ -0,0 +1,51 @@
+﻿/// <reference path="../../../../../typings/tsd.d.ts"/>
+import periodicBackupConfiguration = require("models/database/tasks/periodicBackup/periodicBackupConfiguration");
+
+class serverWideBackupConfiguration extends periodicBackupConfiguration {
+    
+    databasesToExclude = ko.observableArray<string>();
+
+    constructor(databaseName: KnockoutObservable<string>,
+                dto: Raven.Client.ServerWide.Operations.Configuration.ServerWideBackupConfiguration, 
+                serverLimits: periodicBackupServerLimitsResponse, 
+                encryptedDatabase: boolean,
+        isServerWide: boolean) {
+
+        super(databaseName, dto, serverLimits, encryptedDatabase, isServerWide);
+    }
+
+    toDto(): Raven.Client.ServerWide.Operations.Configuration.ServerWideBackupConfiguration {
+        const backupConfigurationDto = super.toDto();
+
+        const dto = backupConfigurationDto as Raven.Client.ServerWide.Operations.Configuration.ServerWideBackupConfiguration;
+        dto.DatabasesToExclude = this.databasesToExclude();
+        return dto;
+    }
+
+    static empty(databaseName: KnockoutObservable<string>, serverLimits: periodicBackupServerLimitsResponse, encryptedDatabase: boolean, isServerWide: boolean): serverWideBackupConfiguration {
+        return new serverWideBackupConfiguration(databaseName, {
```

Comment:
can we get this empty object from parent to avoid code duplicate?

Reply:
done